src/app/modules/shared/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-dialog" role="document">
+  <div class="modal-content" *ngIf="body">
+    <div class="modal-header">
+      <h5 class="modal-title" >{{ title }}</h5>
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+    <div class="modal-body">
+      {{ body }}
+    </div>
+    <div class="modal-footer">
+      <button #cancelDeleteModal type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+      <button (click)="closeModal()" type="button" class="btn btn-danger">Delete</button>
```

Comment:
Replace Delete by Confirm so that this modal can be used by other confirm scenarios